pkg/omitter/reporting_omitter.go

```diff
@@ -0,0 +1,73 @@
+package omitter
+
+import (
+	"sync"
+
+	"github.com/openshift/must-gather-clean/pkg/kube"
+)
+
+type MultiReportingOmitter struct {
+	fileOmitters []FileOmitter
+	k8sOmitters  []KubernetesResourceOmitter
+
+	omittedPathsLock sync.Mutex
+	omittedPaths     []string
+}
+
+func (m *MultiReportingOmitter) OmitPath(path string) (bool, error) {
```

Comment:
Please add a comment/description on what this function does.

Reply:
this is documented on the interface: https://github.com/openshift/must-gather-clean/pull/43/files/dbc7e3c73711cdada169ef2c15dcf9887b97af46#diff-2ba966afd5b579a31a3e3e44617e9a44c92eb64e1f58a8d97ddcaaa546398764R8

`	// OmitPath takes the relative path of the file and its return indicates if the file should be omitted.`